src/app/shared/form-error-messages.component.ts

```diff
@@ -1,55 +1,50 @@
+/**
+ * Centralized component for all form error messages
+ * MUST BE WRAPPED IN A mat-error ELEMENT
+ * Takes a form control as input and outputs a span element with the error message
+ */
+
 import { Component, Input } from '@angular/core';
 import { AbstractControl, AbstractControlDirective } from '@angular/forms';
 
 @Component({
   selector: 'planet-form-error-messages',
   template: `
-    <ul class="text-danger" *ngIf="shouldShowErrors()">
-      <li *ngFor="let error of listOfErrors()">{{error}}</li>
-    </ul>
-  `,
-  styles: [
-    `ul {
-      list-style-type: none;
-      font-size: .8rem;
-      padding-left: .2em;
-    }`
-  ]
+    <span *ngIf="shouldShowError()" i18n>{updateError(), select,
+      required {This field is required}
+      min {The number cannot be below {{number}}}
+      duplicateCourse {This course already exists}
+      duplicateNation {This nation already exists}
```

Comment:
Both error message are the same . It should be different for Nation name and URL.
![screen shot 2017-12-06 at 3 37 44 pm](https://user-images.githubusercontent.com/7495421/33655557-4a2824bc-da6b-11e7-9c43-e5920e5c0912.png)

Reply:
Thanks for catching that.  @lmmrssa and I decided to use a generic duplicate message for all duplication errors at this time to keep things simple.